### ISSUE TITLE
`feature.location.strand` raises NoneType object AttributeError in WebApp

### ISSUE THREAD

**Issue by ilaydagulmez** [moshi4/pyGenomeViz#54]
Hi, first of all thanks for the great tool. I have an issue with Webapp. I have 5 similar gb files but some of them succeed, while others gave an error like this:

<img width="793" alt="Ekran Resmi 2024-07-24 09 40 03" src="https://github.com/user-attachments/assets/edc3eac9-944c-49ad-b07d-b0b84dfe483f">

Any suggestion? 

Best

**Comment by moshi4** [comment]
This seems to be an error that occurs when the location of Feature in Genbank file does not exist, I have never seen a case so far where the location of the Feature does not exist, how did you create your Genbank file?

If you upload the file causing the error, I will try to find out the detailed cause of the error.

**Comment by ilaydagulmez** [comment]
Thanks for reply, I created Genbank file with GeSeq (https://chlorobox.mpimp-golm.mpg.de/geseq.html)

Here is the problematic file:

[896_1_20896_1_GenBank.txt](https://github.com/user-attachments/files/16360823/896_1_20896_1_GenBank.txt)

**Comment by moshi4** [comment]
Inappropriate location information not expected by BioPython's Genbank parser is listed in the Genbank file.
Therefore, `feature.location` is not set correctly and is defined as None, resulting in an AttributeError.

**BioPython location parse warning message**

```
BiopythonParserWarning: Could not parse feature location 'join(238218..23824'; setting feature location to None.
BiopythonParserWarning: Attempting to fix invalid location '35273..73' as it looks like incorrect origin wrapping. Please fix input file, this could have unintended behavior.
BiopythonParserWarning: Attempting to fix invalid location '145659..17173' as it looks like incorrect origin wrapping. Please fix input file, this could have unintended behavior.
```

I will release v1.2.2 which implements a workaround for this issue over the weekend or so.

**Comment by ilaydagulmez** [comment]
Hi, thanks for your help!

**Comment by moshi4** [comment]
Fixed in [v1.3.0](https://github.com/moshi4/pyGenomeViz/releases/tag/v1.3.0) release.
